src/index_with_elements.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Perspective Authorship Demo Using Angular elements</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500|Roboto+Mono:300,400,500|Roboto+Slab|Source+Serif+Pro" rel="stylesheet">
+    <script src='https://www.google.com/recaptcha/api.js?render=explicit' async defer></script>
```

Comment:
Is the idea that these top level scripts need to be added? I think the docs here should aim to explain what is needed to plugin the (remote) component into your webpage, and if these are required it should say that too. If they are not required, we should remove them from the demo?

Looks to me like we are not quite there yet, so maybe all this is to say we should add a TODO to the docs in the README?

Reply:
These are unrelated to the compiled js and are a copy/paste from the other html. I actually have to see which of them we actually still need.

This is not the same as the version we'll be giving clients, because we'll need to give them compiled js where we're still using webpack/whatever angular-cli uses to build this.

Reply:
Yeah, these are incredibly outdated, even for the main demo. Recaptcha was way back in fall 2016 when I was experimenting with the old recaptcha stuff. We don't use platform.js, and I api.js is only used for when we make api requests with gapi. I can delete the first two, but we still need api.js. It's unclear how that will work when we package this up.

Out of curiosity I looked way back and apparently you were the one who added platform.js and api.js originally; not sure what the context was then, maybe you remember? https://github.com/Jigsaw-Code/conversationai-checker/commit/0841ad1bf619a5207baaaa5d7df1ca5a160e2241